markdown/stencil-docs/reference-docs/other-objects-and-properties-overview.md

```diff
@@ -2819,6 +2819,10 @@ https://github.com/bigcommerce/cornerstone/tree/master/templates/components/comm
     <td>recaptcha.markup</td>
     <td>Adds reCaptcha V2</td>
   </tr>
+  <tr>
+  <td>error</td>
+  <td>BC-defined message to display when customer’s login fails</td>

```

Comment:
We never use BC publicly. Always BigCommerce. If you do see it in use, fix it.

Reply:
I also noticed that the form object returns a more information. Please edit the object to include it or create a spin off ticket. 

```
  "forms": {
    "login": {
      "error": "",
      "recaptcha": {
        "enabled": 1,
        "markup": "<div class=\"g-recaptcha\" data-sitekey=\"6LccmasUAAAAAIRhScC9asOrH_rQblw06weNOzDI\"></div><br/>",
        "public_key": "6LdWfFgSABBAAAI83aRectJhbwidegZKk8PzWBltH"
      },
      "success": ""
    }
  },
```

Reply:
`http://localhost:3000/login.php?debug=bar`